uma/utils/cert_utils.go

```diff
@@ -0,0 +1,112 @@
+package utils
+
+import (
+	"crypto/x509/pkix"
+	"encoding/asn1"
+	"encoding/hex"
+	"encoding/pem"
+	"errors"
+	"github.com/decred/dcrd/dcrec/secp256k1/v4"
+	"math/big"
+	"time"
+)
+
+func ConvertPemCertificateChainToHexEncodedDer(certChain *string) ([]string, error) {
+	if certChain == nil {
+		return []string{}, nil
```

Comment:
hmm... I'm trying to think through the implications (if any) of passing an empty array here instead of nil. Because the MarshalJson function uses this, would we hit any weirdness in other language SDKs with an empty array here? I think it might be fine, but we'll find out it testing. Just some food for thought.